packages/console/src/console-history/SizeUsageUI.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../../components/scss/custom.scss';
+
+.max-memory {
+  position: relative;
+  background-color: $gray-900;
+  width: 4rem;
+  vertical-align: middle;
+  text-align: center;
+  border: $black;
+  z-index: 1;
```

Comment:
add user-select: none, so we have default cursor, and 4.00GB isn't text-selectable.